### PR TITLE
First Release

### PR DESCRIPTION
**Pull request by aldemeery**
### Added

- Basic Layout html.
- Content layout.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,6 +11,413 @@
         <link rel="stylesheet" href="./assets/css/style.css" />
     </head>
     <body>
-        <div class="container-fluid"></div>
+        <div class="container-fluid">
+            <nav class="navbar col-12"></nav>
```
Comment (lucasmazo32) [inline index.html:15]: - [ ] There should be elements inside the navbar. As the rest of the project, content is not important.

File: index.html [hunk 2]
```diff
@@ -11,6 +11,413 @@
...
+            </div>
+
+            <div class="bottom-link col-12">
+                One more thing from here as well...
+            </div>
+
+            <footer class="footer col-12">
+                <div class="top-content"></div>
+                <div class="bottom-content"></div>
+            </footer>
```
Comment (lucasmazo32) [inline index.html:420]: - [ ] The footer should have elements like the original page. As the rest of the project, content is not important.

**Comment by aldemeery** [comment]
Hi @lucasmazo32 
Thanks for your response, however it was frustrating for me to read it after waiting for all this time.

Although I might see where your point is coming from about the last two issues (which I don't agree with by the way, and I will elaborate in a moment), I am very surprised by your comments about what should be put in the `.gitignore` file:
- **Changelog**: Changelogs are meant to be for the people out there viewing the repository, so they can know what has been done with each release. Thus they can't be in `.gitignore` files.
- **Prettier file**: `.prettierrc` is the config file we use to enforce a consistent coding style along with stylelint, it must be there to tell anyone who adds new code, how we style our code. 
- `package-lock.json`: Well...it is even mentioned in the **npm** official repository that  **IT IS INTENDED TO BE COMMITTED!**, please check it out [here](https://github.com/npm/npm/blob/v5.0.0/doc/files/package-lock.json.md)
  > package-lock.json is automatically generated for any operations where npm modifies either the node_modules tree, or package.json. It describes the exact tree that was generated, such that subsequent installs are able to generate identical trees, regardless of intermediate dependency updates.
  >
  > **_This file is intended to be committed into source repositories, and serves various purposes:_**

Finally, while content is not important like you said, this is not a **represent-website-using-colored-blocks** project, it is a grayscale heatmap, which is supposed to give an idea about how the user looks at the page, in what direction, and in what order. so unless you think that users look at every single element in the navbar AND in the footer, I don't think that it is mandatory to have blocks representing that. This is still arguable though.

Also, I don't know if you noticed, but the footer (because it is the least part of the page that draws the user attention) is completely white, which means even if we put elements inside it, they won't be visible, because we can't get any colors brighter than white, and elements inside the footer cannot be noticeable more than the footer itself, hence they cannot be darker.

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Comment by aldemeery** [comment]
Issues fixed.

**Pull request merged**